model/BUILD.bazel

```diff
@@ -7,6 +7,13 @@ The java_lite_proto_library() rule takes in a proto_library target and generates
 '''
 load("//model:src/main/proto/format_import_proto_library.bzl", "format_import_proto_library")
 
+# NOTE TO DEVELOPERS: When adding new protos, each proto will need to have both a proto_library
+# and java_lite_proto_library. See the examples below for context. Further, once the proto lite
+# library is added, it should be included in the exports list in the model library at the
+# bottom of this file so that other parts of the app get access to it. If protos import other
+# protos, they need to use format_import_proto_library (again, see examples below for how to do
+# this).
+
```

Comment:
maybe an explicit example could be added to make it a bit simpler to understand otherwise looks good to me

Reply:
I thought about that, but I thought that the existing libraries were simple enough to follow as an example. Do you think an additional example is really needed @aggarwalpulkit596?

Reply:
I think for newcomers bazel would be something very different and they won't be familiar with it in the first place and that's the reason I was thinking to make a simple sample to help new contributors on board easily. We could make a sample repo as well for this and point that repo, if that sounds good to you, @anandwana001 @rt4914 your thoughts?

Reply:
Ah, in that case I agree that we should have more Bazel onboarding instructions as we move farther along (@miaboloix as well for visibility & in case you have any thoughts). That being said, do you think anything additional is needed here for protos? It seemed straightforward since the other libraries below are quite simple (and the example would more or less be exactly the same thing), but if you and/or others think an explicit example is still helpful then I'm happy to add one.

Reply:
I think an explicit example will be helpful.

Reply:
Added an example. @anandwana001 PTAL.

Reply:
Looks helpful to me, Thanks.

Reply:
Looks good to me too, Thanks.